octoprint_mrbeam/iobeam/dust_manager.py

```diff
@@ -513,13 +517,116 @@ def _validate_values(self):
                 rpm=self._rpm,
                 dust=self._dust,
                 connected=self._connected,
-                age=(time.time() - self._data_ts),
+                age=(monotonic_time() - self._data_ts),
             )
             self._pause_laser(trigger="Air filter not connected.", log_message=msg)
 
         # TODO: check for error case in connected val (currently, connected == True/False/None)
+        self._check_and_report_error()
         return result
 
+    def _check_if_one_value_is_none(self):
+        """
+        Checks if one of the values is None and sets the _fan_data_missing_ts flag.
+
+        Returns: (bool, list): True if one of the values is None, False otherwise. And List of error messages.
+        """
+        errs = []
+        result = False
+        if self._state is None or self._rpm is None or self._dust is None:
+            result = True
```

Comment:
For readability (optional):
```suggestion
        has_none_value = any(value is None for value in (self._state, self._rpm, self._dust))
        if has_none_value:
```

Reply:
I will leave it as it was